src/libponyrt/lang/stat.c

```diff
@@ -69,6 +71,8 @@ static void windows_stat(const char* path, pony_stat_t* p, struct __stat64* st,
   p->broken = false;
 
   p->hard_links = (uint32_t)st->st_nlink;
+  p->device = (uint32_t)st->st_dev;
```

Comment:
We discussed this on the sync call, and agreed that we should use `uint64_t`/`U64`.